examples/gallery/3d_plots/scatter3d.py

```diff
@@ -68,4 +73,8 @@
     # Vertical exaggeration factor
     zscale=1.5,
 )
+
+# Add colorbar legend
+fig.colorbar()
```

Comment:
![image](https://user-images.githubusercontent.com/29518865/117768142-dc0a5500-b229-11eb-85c1-33508fd140c8.png)

The colorbar is offset when I view it; my recommendation is to either center it or use `perspective` to make it align with one of the axes.